### PR TITLE
Add sandbox mode and Shipping class

### PR DESCRIPTION
**Pull request by ivan-mosiev-altexsoft**
Added UPS_SANDBOX to env variables that will be passed to UPS classes constructor as 4th parameter. It allows to test UPS integration before go to production. 
Sandbox mode is especially important for Shipping class that allows create shipments and print labels, because this operation require payments in production environment.
Minor code refactoring done in provider class because lines with 4 params were too long

### CODE REVIEW

**Comment by ivan-mosiev-altexsoft** [comment]
Fixed CI notices

File: config/ups.php [hunk 1]
```diff
@@ -14,4 +14,5 @@
     'access_key' => env('UPS_ACCESS_KEY', 'test'),
     'user_id'    => env('UPS_USER_ID', 'test'),
     'password'   => env('UPS_PASSWORD', 'test'),
+    'sandbox'   => env('UPS_SANDBOX', 'true') == 'true',
```
Comment (ptondereau) [inline config/ups.php]: why `== 'true'` ?

**Comment by ptondereau** [comment]
Thank you for your PR !

I've made a comment about it. Can you check it ?

File: config/ups.php (hunk 1 above)
Comment (ivan-mosiev-altexsoft) [inline config/ups.php]: As I understand all envs are strings, that's why basicly.

Comment (ptondereau) [inline config/ups.php]: When you get it from Laravel helper it'll be automatically converted. See https://github.com/laravel/framework/blob/5.3/src/Illuminate/Foundation/helpers.php#L428

**Comment by ptondereau** [comment]
Don't care about Style CI and Nitpick I'll fix up myself. Just check Travis :)

File: config/ups.php (hunk 1 above)
Comment (ivan-mosiev-altexsoft) [inline config/ups.php]: Ok, thank you, I have not so much exp with Laravel, 1 sec will fix

**Comment by ptondereau** [comment]
Seems good to me. Thank you!

**Pull request merged**